test/services/conditionsServiceTest.js

```diff
@@ -175,4 +175,219 @@ describe('licenceDetailsService', () => {
         });
 
     });
+
+    describe('getAdditionalConditionsWithErrors', () => {
+        it('should populate the user input with form data', () => {
+            licenceClient.getAdditionalConditions.resolves([
+                {
+                    ID: {value: 'a'},
+                    TEXT: {value: 'v'},
+                    USER_INPUT: {},
+                    GROUP_NAME: {value: 'g1'},
+                    SUBGROUP_NAME: {value: 's1'},
+                    FIELD_POSITION: {value: {address1: '0', address2: '1'}}
+                }
+            ]);
+
+            const input = {
+                additionalConditions: 'a',
+                address1: '5 Fleet Street',
+                address2: 'London'
+            };
+
+            const validationObject = {
+                validates: false,
+                missing: ['address1']
```

Comment:
I'd probably go with 
```
{
  errors: {
    field1: ["missing"],
    field2: ["toolong"],
    // etc
  }
}
```

to make iteration easier later

Reply:
Agreed